sdk/Trace/Sampler/AlwaysOnSampler.php

```diff
@@ -39,7 +39,7 @@ public function shouldSample(
 
         return new SamplingResult(
             SamplingResult::RECORD_AND_SAMPLE,
-            null,
+            $attributes,
```

Comment:
This is so that the attributes passed to the samper are propagated to the created span. Not sure why this was `null` in the first place.

Reply:
Good question.  I'm not sure why either.